---
title: "RNASeq data screening"
output:
  md_document:
    variant: markdown_github
---

```{r}
source("https://bioconductor.org/biocLite.R")
biocLite("qrqc")
library(qrqc)
library(ggplot2)
getwd()
```

```{r}
fqfiles.43.R1 <- c(none="../data/CP-4Spl_S11_L004_R1_0343.fastq",
             sickle="../data/CP-4Spl_S11_L004_R1_0343_sickle.fastq",
             trimfq="../data/CP-4Spl_S11_L004_R1_0343_trimfq.fastq")

fqfiles.43.R2 <- c(none="../data/CP-4Spl_S11_L004_R2_0343.fastq",
             sickle="../data/CP-4Spl_S11_L004_R2_0343_sickle.fastq",
             trimfq="../data/CP-4Spl_S11_L004_R2_0343_trimfq.fastq")
```

Load each file in, using qrqc's readSeqFile
We only need qualities, so we turn off some of readSeqFile's other features.

```{r}
seq_info.43.R1 <- lapply(fqfiles.43.R1, function(file) {
          readSeqFile(file, hash=FALSE, kmer=FALSE)
})
seq_info.43.R2 <- lapply(fqfiles.43.R2, function(file) {
          readSeqFile(file, hash=FALSE, kmer=FALSE)
})
```

Extract the qualities as dataframe, and append a column of which trimmer (or none) was used. This is used in later plots.

```{r}
quals.43.R1 <- mapply(function(sfq, name) {
          qs <- getQual(sfq)
          qs$trimmer <- name
          qs
}, seq_info.43.R1, names(fqfiles.43.R1), SIMPLIFY = FALSE)

quals.43.R2 <- mapply(function(sfq, name) {
          qs <- getQual(sfq)
          qs$trimmer <- name
          qs
}, seq_info.43.R2, names(fqfiles.43.R2), SIMPLIFY = FALSE)
```

Combine separate dataframes in a list into single dataframe

```{r}
d.43.R1 <- do.call(rbind, quals.43.R1)
d.43.R2 <- do.call(rbind, quals.43.R2)
```

Visualize qualities

```{r}
p1 <- ggplot(d.43.R1) + geom_line(aes(x=position, y=mean, linetype=trimmer))
p1 <- p1 + ylab("mean quality (sanger)") + theme_bw()
print(p1)

p2 <- ggplot(d.43.R2) + geom_line(aes(x=position, y=mean, linetype=trimmer))
p2 <- p2 + ylab("mean quality (sanger)") + theme_bw()
print(p2)
```
 
## Repeat with the other 2 data files, which are from the other lane: 

```{r}
fqfiles.48.R1 <- c(none="../data/CP-4Spl_S11_L004_R1_0348.fastq",
             sickle="../data/CP-4Spl_S11_L004_R1_0348_sickle.fastq",
             trimfq="../data/CP-4Spl_S11_L004_R1_0348_trimfq.fastq")

fqfiles.48.R2 <- c(none="../data/CP-4Spl_S11_L004_R2_0348.fastq",
             sickle="../data/CP-4Spl_S11_L004_R2_0348_sickle.fastq",
             trimfq="../data/CP-4Spl_S11_L004_R2_0348_trimfq.fastq")

seq_info.48.R1 <- lapply(fqfiles.48.R1, function(file) {
          readSeqFile(file, hash=FALSE, kmer=FALSE)
})
seq_info.48.R2 <- lapply(fqfiles.48.R2, function(file) {
          readSeqFile(file, hash=FALSE, kmer=FALSE)
})

quals.48.R1 <- mapply(function(sfq, name) {
          qs <- getQual(sfq)
          qs$trimmer <- name
          qs
}, seq_info.48.R1, names(fqfiles.48.R1), SIMPLIFY = FALSE)

quals.48.R2 <- mapply(function(sfq, name) {
          qs <- getQual(sfq)
          qs$trimmer <- name
          qs
}, seq_info.48.R2, names(fqfiles.48.R2), SIMPLIFY = FALSE)

d.48.R1 <- do.call(rbind, quals.48.R1)
d.48.R2 <- do.call(rbind, quals.48.R2)

p3 <- ggplot(d.48.R1) + geom_line(aes(x=position, y=mean, linetype=trimmer))
p3 <- p3 + ylab("mean quality (sanger)") + theme_bw()
print(p3)

p4 <- ggplot(d.48.R2) + geom_line(aes(x=position, y=mean, linetype=trimmer))
p4 <- p4 + ylab("mean quality (sanger)") + theme_bw()
print(p4)
```

