{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/Users/laura/Documents/roberts-lab/laura-quantseq/notebooks'"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "### Installed Trinity vs2.8.3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "trinity = \"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "#########################################################################\r\n",
      "#\r\n",
      "########################\r\n",
      "#  Essential parameters:\r\n",
      "########################\r\n",
      "#\r\n",
      "#  --transcripts <string>           transcript fasta file\r\n",
      "#\r\n",
      "#  --seqType <string>               fq|fa\r\n",
      "# \r\n",
      "#  If Paired-end:\r\n",
      "#\r\n",
      "#     --left <string>\r\n",
      "#     --right <string>\r\n",
      "#  \r\n",
      "#   or Single-end:\r\n",
      "#\r\n",
      "#      --single <string>\r\n",
      "#\r\n",
      "#   or (preferred):\r\n",
      "#\r\n",
      "#      --samples_file <string>    tab-delimited text file indicating biological replicate relationships.\r\n",
      "#                                   ex.\r\n",
      "#                                        cond_A    cond_A_rep1    A_rep1_left.fq    A_rep1_right.fq\r\n",
      "#                                        cond_A    cond_A_rep2    A_rep2_left.fq    A_rep2_right.fq\r\n",
      "#                                        cond_B    cond_B_rep1    B_rep1_left.fq    B_rep1_right.fq\r\n",
      "#                                        cond_B    cond_B_rep2    B_rep2_left.fq    B_rep2_right.fq\r\n",
      "#\r\n",
      "#                      # if single-end instead of paired-end, then leave the 4th column above empty.\r\n",
      "#\r\n",
      "#\r\n",
      "#\r\n",
      "#  --est_method <string>           abundance estimation method.\r\n",
      "#                                        alignment_based:  RSEM|eXpress       \r\n",
      "#                                        alignment_free: kallisto|salmon\r\n",
      "#  \r\n",
      "###################################\r\n",
      "#  Potentially optional parameters:\r\n",
      "###################################\r\n",
      "#\r\n",
      "# --output_dir <string>            write all files to output directory \r\n",
      "#                                  (note, if using --samples_file, output_dir will be set automatically according to replicate name))\r\n",
      "#  \r\n",
      "#\r\n",
      "#  if alignment_based est_method:\r\n",
      "#       --aln_method <string>            bowtie|bowtie2 alignment method.  (note: RSEM requires bowtie)\r\n",
      "#                                       \r\n",
      "###########\r\n",
      "# Optional:\r\n",
      "# #########\r\n",
      "#\r\n",
      "# --SS_lib_type <string>           strand-specific library type:  paired('RF' or 'FR'), single('F' or 'R').\r\n",
      "#                                  \r\n",
      "# --samples_idx <int>               restricte processing to sample entry (index starts at one)\r\n",
      "#\r\n",
      "#\r\n",
      "# --thread_count                   number of threads to use (default = 4)\r\n",
      "#\r\n",
      "# --debug                          retain intermediate files\r\n",
      "#\r\n",
      "#  --gene_trans_map <string>        file containing 'gene(tab)transcript' identifiers per line.\r\n",
      "#     or  \r\n",
      "#  --trinity_mode                   Setting --trinity_mode will automatically generate the gene_trans_map and use it.\r\n",
      "#\r\n",
      "#\r\n",
      "#  --prep_reference                 prep reference (builds target index)\r\n",
      "#\r\n",
      "#\r\n",
      "########################################\r\n",
      "#\r\n",
      "#  Parameters for single-end reads:\r\n",
      "#\r\n",
      "#  --fragment_length <int>         specify RNA-Seq fragment length (default: 200) \r\n",
      "#  --fragment_std <int>            fragment length standard deviation (defalt: 80)\r\n",
      "#\r\n",
      "########################################\r\n",
      "#  \r\n",
      "#   bowtie-related parameters: (note, tool-specific settings are further below)\r\n",
      "#\r\n",
      "#  --max_ins_size <int>             maximum insert size (bowtie -X parameter, default: 800)\r\n",
      "#  --coordsort_bam                  provide coord-sorted bam in addition to the default (unsorted) bam.\r\n",
      "#\r\n",
      "########################################\r\n",
      "#  RSEM opts:\r\n",
      "#\r\n",
      "#  --bowtie_RSEM <string>          if using 'bowtie', default: \"--all --best --strata -m 300 --chunkmbs 512\"\r\n",
      "#  --bowtie2_RSEM <string>         if using 'bowtie2', default: \"--no-mixed --no-discordant --gbar 1000 --end-to-end -k 200 \"\r\n",
      "#                                ** if you change the defaults, specify the full set of parameters to use! **\r\n",
      "#\r\n",
      "#  --include_rsem_bam              provide the RSEM enhanced bam file including posterior probabilities of read assignments.\r\n",
      "#  --rsem_add_opts <string>        additional parameters to pass on to rsem-calculate-expression\r\n",
      "#\r\n",
      "##########################################################################\r\n",
      "#  eXpress opts:\r\n",
      "#\r\n",
      "#  --bowtie_eXpress <string>  default: \"--all --best --strata -m 300 --chunkmbs 512\"\r\n",
      "#  --bowtie2_eXpress <string> default: \"--no-mixed --no-discordant --gbar 1000 --end-to-end -k 200 \"\r\n",
      "#  --eXpress_add_opts <string>  default: \"\"\r\n",
      "#\r\n",
      "##########################################################################\r\n",
      "#  kallisto opts:\r\n",
      "#\r\n",
      "#  --kallisto_add_opts <string>  default:   \r\n",
      "#\r\n",
      "##########################################################################\r\n",
      "#\r\n",
      "#  salmon opts:\r\n",
      "#\r\n",
      "#  --salmon_idx_type <string>    quasi|fmd (defalt: quasi)\r\n",
      "#  --salmon_add_opts <string>    default: \r\n",
      "#\r\n",
      "#\r\n",
      "#  Example usage\r\n",
      "#\r\n",
      "#   ## Just prepare the reference for alignment and abundance estimation\r\n",
      "#\r\n",
      "#    /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/align_and_estimate_abundance.pl --transcripts Trinity.fasta --est_method RSEM --aln_method bowtie --trinity_mode --prep_reference\r\n",
      "#\r\n",
      "#   ## Run the alignment and abundance estimation (assumes reference has already been prepped, errors-out if prepped reference not located.)\r\n",
      "#\r\n",
      "#    /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/align_and_estimate_abundance.pl --transcripts Trinity.fasta --seqType fq --left reads_1.fq --right reads_2.fq --est_method RSEM --aln_method bowtie --trinity_mode --output_dir rsem_outdir\r\n",
      "#\r\n",
      "##  ## prep the reference and run the alignment/estimation\r\n",
      "#\r\n",
      "#    /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/align_and_estimate_abundance.pl --transcripts Trinity.fasta --seqType fq --left reads_1.fq --right reads_2.fq --est_method RSEM --aln_method bowtie --trinity_mode --prep_reference --output_dir rsem_outdir\r\n",
      "#\r\n",
      "#   ## Use a samples.txt file:\r\n",
      "#\r\n",
      "#    /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/align_and_estimate_abundance.pl --transcripts Trinity.fasta --est_method RSEM --aln_method bowtie2 --prep_reference --trinity_mode --samples_file samples.txt --seqType fq  \r\n",
      "#\r\n",
      "#########################################################################\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "# check that the shortcut is working, and review the perl script \n",
    "! {trinity}util/align_and_estimate_abundance.pl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../results/kallisto/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "CMD: kallisto index -i /Users/laura/Documents/roberts-lab/laura-quantseq/notebooks/../results/trinity/20181029-olurida-gonad-assembly.fasta.kallisto_idx /Users/laura/Documents/roberts-lab/laura-quantseq/notebooks/../results/trinity/20181029-olurida-gonad-assembly.fasta\n",
      "\n",
      "[build] loading fasta file /Users/laura/Documents/roberts-lab/laura-quantseq/notebooks/../results/trinity/20181029-olurida-gonad-assembly.fasta\n",
      "[build] k-mer length: 31\n",
      "[build] warning: clipped off poly-A tail (longer than 10)\n",
      "        from 1146 target sequences\n",
      "[build] counting k-mers ... done.\n",
      "[build] building target de Bruijn graph ...  done \n",
      "[build] creating equivalence classes ...  done\n",
      "[build] target de Bruijn graph has 565703 contigs and contains 72762537 k-mers \n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/notebooks/../results/trinity/20181029-olurida-gonad-assembly.fasta.kallisto_idx -o ../results/kallisto/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/notebooks/../data/tagseq_trim/CP-KS-LibL-L10-G_S59_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 181,531\n",
      "[index] number of k-mers: 72,762,537\n",
      "[index] number of equivalence classes: 397,582\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/notebooks/../data/tagseq_trim/CP-KS-LibL-L10-G_S59_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 21,874 reads, 8,585 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 620 rounds\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/align_and_estimate_abundance.pl \\\n",
    "--transcripts ../results/trinity/20181029-olurida-gonad-assembly_clust.fasta \\\n",
    "--prep_reference \\\n",
    "--seqType fq \\\n",
    "--single ../data/tagseq_trim/CP-KS-LibL-L10-G_S59_L006_R1_001.fastq.trim \\\n",
    "--output_dir ../results/kallisto/ \\\n",
    "--est_method kallisto"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "target_id\tlength\teff_length\test_counts\ttpm\r\n",
      "TRINITY_DN88414_c0_g1_i1\t228\t74.1334\t0\t0\r\n",
      "TRINITY_DN88428_c0_g1_i1\t286\t106.21\t0\t0\r\n",
      "TRINITY_DN88485_c0_g1_i1\t493\t292.649\t0\t0\r\n",
      "TRINITY_DN88461_c0_g1_i1\t305\t119.325\t0\t0\r\n",
      "TRINITY_DN88458_c0_g1_i1\t843\t642.611\t0\t0\r\n",
      "TRINITY_DN88473_c0_g1_i1\t232\t75.9808\t0\t0\r\n",
      "TRINITY_DN88442_c0_g1_i1\t444\t243.911\t0\t0\r\n",
      "TRINITY_DN88493_c0_g1_i1\t865\t664.611\t0\t0\r\n",
      "TRINITY_DN88449_c0_g1_i1\t270\t96.1797\t0\t0\r\n"
     ]
    }
   ],
   "source": [
    "! head ../results/kallisto/abundance.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "####################################################################################\r\n",
      "#\r\n",
      "# Usage:  /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl --est_method <method>  sample1.results sample2.results ...\r\n",
      "#\r\n",
      "#      or  /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl --est_method <method> --quant_files file.listing_target_files.txt\r\n",
      "#\r\n",
      "#      Note, if only a single input file is given, it's expected to contain the paths to all the target abundance estimation files.\r\n",
      "#\r\n",
      "# Required:\r\n",
      "#            \r\n",
      "#  --est_method <string>           RSEM|eXpress|kallisto|salmon  (needs to know what format to expect)\r\n",
      "#\r\n",
      "#  --gene_trans_map <string>           the gene-to-transcript mapping file. (if you don't want gene estimates, indicate 'none'.\r\n",
      "#\r\n",
      "#\r\n",
      "# Options:\r\n",
      "#\r\n",
      "#  --cross_sample_norm <string>         TMM|UpperQuartile|none   (default: TMM)\r\n",
      "#\r\n",
      "#  --name_sample_by_basedir             name sample column by dirname instead of filename\r\n",
      "#      --basedir_index <int>            default(-2)\r\n",
      "#\r\n",
      "#  --out_prefix <string>                default: value for --est_method\r\n",
      "#\r\n",
      "#  --quant_files <string>              file containing a list of all the target files.\r\n",
      "#\r\n",
      "######################################################################################\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "! /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto\n"
     ]
    }
   ],
   "source": [
    "cd ../results/kallisto/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "-reading file: abundance.tsv\n",
      "Use of uninitialized value $column_header in substitution (s///) at /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl line 232.\n",
      "Use of uninitialized value $column_header in hash element at /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl line 234.\n",
      "\n",
      "\n",
      "* Outputting combined matrix.\n",
      "\n",
      "Use of uninitialized value in list assignment at /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl line 245.\n",
      "Use of uninitialized value $filenames[0] in join or string at /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl line 259.\n",
      "Use of uninitialized value $filenames[0] in join or string at /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl line 260.\n",
      "Warning, only one sample, so not performing cross-sample normalization\n",
      "Done.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl \\\n",
    "--est_method kallisto \\\n",
    "--out_prefix matrix1933-trans_counts \\\n",
    "--name_sample_by_basedir \\\n",
    "--gene_trans_map none \\\n",
    "abundance.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "abundance.h5\r\n",
      "abundance.tsv\r\n",
      "matrix1933-trans_counts.isoform.TPM.not_cross_norm\r\n",
      "matrix1933-trans_counts.isoform.counts.matrix\r\n",
      "run_info.json\r\n"
     ]
    }
   ],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t\r\n",
      "TRINITY_DN11903_c0_g1_i8\t0\r\n",
      "TRINITY_DN100904_c0_g1_i1\t0\r\n",
      "TRINITY_DN3344_c0_g1_i3\t0\r\n",
      "TRINITY_DN72677_c0_g1_i1\t0\r\n",
      "TRINITY_DN2169_c0_g1_i8\t0\r\n",
      "TRINITY_DN20110_c0_g1_i1\t0\r\n",
      "TRINITY_DN4284_c0_g1_i1\t0\r\n",
      "TRINITY_DN102019_c0_g1_i1\t0\r\n",
      "TRINITY_DN5989_c1_g1_i1\t0\r\n"
     ]
    }
   ],
   "source": [
    "! head matrix1933-trans_counts.isoform.counts.matrix"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Write loop to do the above on all samples "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/tagseq_trim/CP-KS-LibL-L10-G_S59_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L11-G_S65_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L13-G_S60_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L14-G_S63_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L15-G_S68_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L19-G_S67_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L2-G_S34_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L21-G_S58_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L23-G_S61_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L24-G_S62_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L29-G_S36_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L3-G_S64_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L32-G_S35_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L33-G_S66_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L34-L_S25_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L35-L_S31_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L37-L_S32_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L39-L_S29_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L41-L_S27_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L44-L_S30_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L45-L_S28_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L47-L_S33_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-L7-G_S57_L006_R1_001.fastq.trim\r\n",
      "../../data/tagseq_trim/CP-KS-LibL-SN1016-G_S26_L006_R1_001.fastq.trim\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/tagseq_trim/*.trim"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run kallisto on all trimmed quantseq files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CP-KS-LibL-L10-G_S59\r\n",
      "CP-KS-LibL-L11-G_S65\r\n",
      "CP-KS-LibL-L13-G_S60\r\n",
      "CP-KS-LibL-L14-G_S63\r\n",
      "CP-KS-LibL-L15-G_S68\r\n",
      "CP-KS-LibL-L19-G_S67\r\n",
      "CP-KS-LibL-L2-G_S34\r\n",
      "CP-KS-LibL-L21-G_S58\r\n",
      "CP-KS-LibL-L23-G_S61\r\n",
      "CP-KS-LibL-L24-G_S62\r\n",
      "CP-KS-LibL-L29-G_S36\r\n",
      "CP-KS-LibL-L3-G_S64\r\n",
      "CP-KS-LibL-L32-G_S35\r\n",
      "CP-KS-LibL-L33-G_S66\r\n",
      "CP-KS-LibL-L34-L_S25\r\n",
      "CP-KS-LibL-L35-L_S31\r\n",
      "CP-KS-LibL-L37-L_S32\r\n",
      "CP-KS-LibL-L39-L_S29\r\n",
      "CP-KS-LibL-L41-L_S27\r\n",
      "CP-KS-LibL-L44-L_S30\r\n",
      "CP-KS-LibL-L45-L_S28\r\n",
      "CP-KS-LibL-L47-L_S33\r\n",
      "CP-KS-LibL-L7-G_S57\r\n",
      "CP-KS-LibL-SN1016-G_S26\r\n"
     ]
    }
   ],
   "source": [
    "! ls ../../data/tagseq_trim/*.trim | xargs basename -s _L006_R1_001.fastq.trim"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L10-G_S59_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L10-G_S59_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L10-G_S59_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 21,874 reads, 8,364 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 621 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L11-G_S65_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L11-G_S65_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L11-G_S65_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 21,768 reads, 7,536 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 615 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L13-G_S60_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L13-G_S60_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L13-G_S60_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 2,649 reads, 319 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 361 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L14-G_S63_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L14-G_S63_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L14-G_S63_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 21,489 reads, 7,533 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 621 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L15-G_S68_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L15-G_S68_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L15-G_S68_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 23,903 reads, 8,401 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 647 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L19-G_S67_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L19-G_S67_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L19-G_S67_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 22,552 reads, 7,831 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 507 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L2-G_S34_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L2-G_S34_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L2-G_S34_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 16,370 reads, 6,037 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 556 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L21-G_S58_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L21-G_S58_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L21-G_S58_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 15,257 reads, 5,534 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 519 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L23-G_S61_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L23-G_S61_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L23-G_S61_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 19,906 reads, 6,841 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 586 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L24-G_S62_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L24-G_S62_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L24-G_S62_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 13,103 reads, 5,270 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 607 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L29-G_S36_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L29-G_S36_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L29-G_S36_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 21,913 reads, 7,086 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 614 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L3-G_S64_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L3-G_S64_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L3-G_S64_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 3,923 reads, 566 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 376 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L32-G_S35_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L32-G_S35_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L32-G_S35_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 23,289 reads, 7,175 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 618 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L33-G_S66_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L33-G_S66_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L33-G_S66_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 23,769 reads, 8,691 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 523 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L34-L_S25_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L34-L_S25_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L34-L_S25_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 3,290 reads, 908 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 548 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L35-L_S31_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L35-L_S31_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L35-L_S31_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 13,007 reads, 3,317 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 599 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L37-L_S32_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L37-L_S32_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L37-L_S32_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 5,448 reads, 1,494 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 586 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L39-L_S29_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L39-L_S29_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L39-L_S29_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 8,559 reads, 2,565 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 548 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L41-L_S27_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L41-L_S27_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L41-L_S27_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 6,827 reads, 1,993 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 548 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L44-L_S30_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L44-L_S30_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L44-L_S30_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 8,856 reads, 3,524 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 575 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L45-L_S28_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L45-L_S28_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L45-L_S28_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 10,630 reads, 3,636 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 592 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L47-L_S33_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L47-L_S33_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L47-L_S33_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 5,586 reads, 1,720 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 519 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-L7-G_S57_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L7-G_S57_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-L7-G_S57_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 23,593 reads, 8,845 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 603 rounds\n",
      "\n",
      "CMD: kallisto quant -l 200 -s 80 -i /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta.kallisto_idx -o CP-KS-LibL-SN1016-G_S26_L006_R1_001/  --single /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-SN1016-G_S26_L006_R1_001.fastq.trim\n",
      "\n",
      "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 80\n",
      "[index] k-mer length: 31\n",
      "[index] number of targets: 143,867\n",
      "[index] number of k-mers: 70,340,116\n",
      "[index] number of equivalence classes: 289,792\n",
      "[quant] running in single-end mode\n",
      "[quant] will process file 1: /Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/../../data/tagseq_trim/CP-KS-LibL-SN1016-G_S26_L006_R1_001.fastq.trim\n",
      "[quant] finding pseudoalignments for the reads ... done\n",
      "[quant] processed 10,786 reads, 3,974 reads pseudoaligned\n",
      "[   em] quantifying the abundances ... done\n",
      "[   em] the Expectation-Maximization algorithm ran for 603 rounds\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "find ../../data/tagseq_trim/*.trim \\\n",
    "| xargs basename -s .fastq.trim | xargs -I{} \\\n",
    "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/align_and_estimate_abundance.pl \\\n",
    "--transcripts ../../results/trinity/20181029-olurida-gonad-assembly_clust.fasta \\\n",
    "--prep_reference \\\n",
    "--seqType fq \\\n",
    "--single ../../data/tagseq_trim/{}.fastq.trim \\\n",
    "--output_dir {}/ \\\n",
    "--est_method kallisto"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The above script uses find and xargs basename -s to search for all .trim files, extract the file name that's not redundant (at least the end of the file name, some redundancy remains in the beginning of the names), then do the pseudoalignment using the perl script, and write the output files to directories that are named using the file name. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "####################################################################################\r\n",
      "#\r\n",
      "# Usage:  /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl --est_method <method>  sample1.results sample2.results ...\r\n",
      "#\r\n",
      "#      or  /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl --est_method <method> --quant_files file.listing_target_files.txt\r\n",
      "#\r\n",
      "#      Note, if only a single input file is given, it's expected to contain the paths to all the target abundance estimation files.\r\n",
      "#\r\n",
      "# Required:\r\n",
      "#            \r\n",
      "#  --est_method <string>           RSEM|eXpress|kallisto|salmon  (needs to know what format to expect)\r\n",
      "#\r\n",
      "#  --gene_trans_map <string>           the gene-to-transcript mapping file. (if you don't want gene estimates, indicate 'none'.\r\n",
      "#\r\n",
      "#\r\n",
      "# Options:\r\n",
      "#\r\n",
      "#  --cross_sample_norm <string>         TMM|UpperQuartile|none   (default: TMM)\r\n",
      "#\r\n",
      "#  --name_sample_by_basedir             name sample column by dirname instead of filename\r\n",
      "#      --basedir_index <int>            default(-2)\r\n",
      "#\r\n",
      "#  --out_prefix <string>                default: value for --est_method\r\n",
      "#\r\n",
      "#  --quant_files <string>              file containing a list of all the target files.\r\n",
      "#\r\n",
      "######################################################################################\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "! /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "-reading file: CP-KS-LibL-L10-G_S59_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L11-G_S65_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L13-G_S60_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L14-G_S63_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L15-G_S68_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L19-G_S67_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L2-G_S34_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L21-G_S58_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L23-G_S61_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L24-G_S62_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L29-G_S36_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L3-G_S64_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L32-G_S35_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L33-G_S66_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L34-L_S25_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L35-L_S31_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L37-L_S32_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L39-L_S29_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L41-L_S27_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L44-L_S30_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L45-L_S28_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L47-L_S33_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-L7-G_S57_L006_R1_001/abundance.tsv\n",
      "-reading file: CP-KS-LibL-SN1016-G_S26_L006_R1_001/abundance.tsv\n",
      "\n",
      "\n",
      "* Outputting combined matrix.\n",
      "\n",
      "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/support_scripts/run_TMM_scale_matrix.pl --matrix 1129matrix-trans_counts.isoform.TPM.not_cross_norm > 1129matrix-trans_counts.isoform.TMM.EXPR.matrixCMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.TPM.not_cross_norm.runTMM.R 1>&2 \n",
      "> library(edgeR)\n",
      "Loading required package: limma\n",
      "> \n",
      "> rnaseqMatrix = read.table(\"1129matrix-trans_counts.isoform.TPM.not_cross_norm\", header=T, row.names=1, com='', check.names=F)\n",
      "> rnaseqMatrix = as.matrix(rnaseqMatrix)\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=factor(colnames(rnaseqMatrix)))\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> exp_study$samples$eff.lib.size = exp_study$samples$lib.size * exp_study$samples$norm.factors\n",
      "> write.table(exp_study$samples, file=\"1129matrix-trans_counts.isoform.TPM.not_cross_norm.TMM_info.txt\", quote=F, sep=\"\\t\", row.names=F)\n",
      "> \n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/util/abundance_estimates_to_matrix.pl \\\n",
    "--est_method kallisto \\\n",
    "--out_prefix 1129matrix-trans_counts \\\n",
    "--name_sample_by_basedir \\\n",
    "--gene_trans_map none \\\n",
    "*_L006_R1_001/abundance.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "$VAR1 = {\n",
      "          'CP-KS-LibL-L35-L_S31_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L34-L_S25_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L7-G_S57_L006_R1_001' => [\n",
      "                                                 'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "                                               ],\n",
      "          'CP-KS-LibL-L23-G_S61_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L32-G_S35_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L44-L_S30_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L37-L_S32_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L11-G_S65_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L11-G_S65_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L45-L_S28_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L21-G_S58_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L15-G_S68_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L15-G_S68_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L14-G_S63_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L14-G_S63_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L2-G_S34_L006_R1_001' => [\n",
      "                                                 'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "                                               ],\n",
      "          'CP-KS-LibL-L33-G_S66_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L41-L_S27_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L24-G_S62_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L13-G_S60_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L13-G_S60_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L10-G_S59_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L10-G_S59_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L3-G_S64_L006_R1_001' => [\n",
      "                                                 'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "                                               ],\n",
      "          'CP-KS-LibL-L29-G_S36_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L47-L_S33_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-SN1016-G_S26_L006_R1_001' => [\n",
      "                                                     'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "                                                   ],\n",
      "          'CP-KS-LibL-L39-L_S29_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "                                                ],\n",
      "          'CP-KS-LibL-L19-G_S67_L006_R1_001' => [\n",
      "                                                  'CP-KS-LibL-L19-G_S67_L006_R1_001'\n",
      "                                                ]\n",
      "        };\n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L11-G_S65_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L11-G_S65_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,2)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L11-G_S65_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L11-G_S65_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L11-G_S65_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L11-G_S65_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L13-G_S60_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,3)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L13-G_S60_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L14-G_S63_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,4)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L14-G_S63_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L15-G_S68_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,5)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L15-G_S68_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L19-G_S67_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,6)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L19-G_S67_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L2-G_S34_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,7)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L2-G_S34_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L10-G_S59_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(1,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L10-G_S59_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L10-G_S59_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L10-G_S59_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L13-G_S60_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,3)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L13-G_S60_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L13-G_S60_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L14-G_S63_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,4)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L14-G_S63_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L15-G_S68_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,5)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L15-G_S68_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L19-G_S67_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,6)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L19-G_S67_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L2-G_S34_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,7)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L2-G_S34_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L11-G_S65_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(2,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L11-G_S65_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L11-G_S65_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L11-G_S65_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L14-G_S63_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,4)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L14-G_S63_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L14-G_S63_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L15-G_S68_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,5)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L15-G_S68_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L19-G_S67_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,6)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L19-G_S67_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L2-G_S34_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,7)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L2-G_S34_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L13-G_S60_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(3,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L13-G_S60_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L13-G_S60_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L13-G_S60_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L15-G_S68_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,5)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L15-G_S68_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L15-G_S68_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L19-G_S67_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,6)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L19-G_S67_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L2-G_S34_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,7)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L2-G_S34_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L14-G_S63_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(4,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L14-G_S63_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L14-G_S63_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L14-G_S63_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L19-G_S67_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,6)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L19-G_S67_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L19-G_S67_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L2-G_S34_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,7)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L2-G_S34_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L15-G_S68_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(5,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L15-G_S68_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L15-G_S68_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L15-G_S68_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L2-G_S34_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,7)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L2-G_S34_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L2-G_S34_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L19-G_S67_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(6,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L19-G_S67_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L19-G_S67_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L19-G_S67_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L21-G_S58_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,8)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L21-G_S58_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L21-G_S58_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L2-G_S34_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(7,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L2-G_S34_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L2-G_S34_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L2-G_S34_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L23-G_S61_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,9)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L23-G_S61_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L23-G_S61_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L21-G_S58_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(8,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L21-G_S58_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L21-G_S58_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L21-G_S58_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L24-G_S62_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,10)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L24-G_S62_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L24-G_S62_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L23-G_S61_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(9,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L23-G_S61_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L23-G_S61_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L23-G_S61_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L29-G_S36_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,11)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L29-G_S36_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L29-G_S36_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L24-G_S62_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(10,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L24-G_S62_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L24-G_S62_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L24-G_S62_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L3-G_S64_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,12)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L3-G_S64_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L3-G_S64_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L29-G_S36_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(11,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L29-G_S36_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L29-G_S36_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L29-G_S36_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L32-G_S35_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,13)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L32-G_S35_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L32-G_S35_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L3-G_S64_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(12,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L3-G_S64_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L3-G_S64_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L3-G_S64_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L33-G_S66_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,14)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L33-G_S66_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L33-G_S66_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L32-G_S35_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(13,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L32-G_S35_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L32-G_S35_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L32-G_S35_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L34-L_S25_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,15)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L34-L_S25_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L34-L_S25_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L33-G_S66_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(14,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L33-G_S66_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L33-G_S66_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L33-G_S66_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L35-L_S31_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,16)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L35-L_S31_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L35-L_S31_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L34-L_S25_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(15,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L34-L_S25_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L34-L_S25_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L34-L_S25_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L37-L_S32_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,17)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L37-L_S32_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L37-L_S32_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L35-L_S31_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(16,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L35-L_S31_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L35-L_S31_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L35-L_S31_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-L39-L_S29_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,18)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-L39-L_S29_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L39-L_S29_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L37-L_S32_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(17,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L37-L_S32_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L37-L_S32_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L37-L_S32_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.CP-KS-LibL-L39-L_S29_L006_R1_001.vs.CP-KS-LibL-L41-L_S27_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(18,19)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1), rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", \"CP-KS-LibL-L41-L_S27_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", sampleB=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L41-L_S27_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L39-L_S29_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(18,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L39-L_S29_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(18,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L39-L_S29_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(18,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L39-L_S29_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(18,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L39-L_S29_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(18,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L39-L_S29_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L39-L_S29_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L39-L_S29_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.CP-KS-LibL-L41-L_S27_L006_R1_001.vs.CP-KS-LibL-L44-L_S30_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(19,20)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1), rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", \"CP-KS-LibL-L44-L_S30_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", sampleB=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L44-L_S30_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L41-L_S27_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(19,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L41-L_S27_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(19,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L41-L_S27_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(19,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L41-L_S27_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(19,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L41-L_S27_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L41-L_S27_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L41-L_S27_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.CP-KS-LibL-L44-L_S30_L006_R1_001.vs.CP-KS-LibL-L45-L_S28_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(20,21)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1), rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", \"CP-KS-LibL-L45-L_S28_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", sampleB=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L45-L_S28_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L44-L_S30_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(20,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L44-L_S30_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(20,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L44-L_S30_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(20,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L44-L_S30_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L44-L_S30_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L44-L_S30_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.CP-KS-LibL-L45-L_S28_L006_R1_001.vs.CP-KS-LibL-L47-L_S33_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(21,22)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1), rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", \"CP-KS-LibL-L47-L_S33_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", sampleB=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L47-L_S33_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L45-L_S28_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(21,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L45-L_S28_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(21,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L45-L_S28_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L45-L_S28_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L45-L_S28_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.CP-KS-LibL-L47-L_S33_L006_R1_001.vs.CP-KS-LibL-L7-G_S57_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(22,23)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1), rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", \"CP-KS-LibL-L7-G_S57_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", sampleB=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-L7-G_S57_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L47-L_S33_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(22,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L47-L_S33_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L47-L_S33_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L47-L_S33_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n",
      "CMD: R --no-save --no-restore --no-site-file --no-init-file -q < 1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L7-G_S57_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.CP-KS-LibL-L7-G_S57_L006_R1_001.vs.CP-KS-LibL-SN1016-G_S26_L006_R1_001.EdgeR.Rscript\n",
      "> if (! require(edgeR)) {\n",
      "+    source(\"https://bioconductor.org/biocLite.R\")\n",
      "+    biocLite(\"edgeR\")\n",
      "+    library(edgeR)\n",
      "+ }\n",
      "> \n",
      "> data = read.table(\"/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto/1129matrix-trans_counts.isoform.counts.matrix\", header=T, row.names=1, com='')\n",
      "> col_ordering = c(23,24)\n",
      "> rnaseqMatrix = data[,col_ordering]\n",
      "> rnaseqMatrix = round(rnaseqMatrix)\n",
      "> rnaseqMatrix = rnaseqMatrix[rowSums(cpm(rnaseqMatrix) > 1) >= 1,]\n",
      "> conditions = factor(c(rep(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", 1), rep(\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", 1)))\n",
      "> \n",
      "> exp_study = DGEList(counts=rnaseqMatrix, group=conditions)\n",
      "> exp_study = calcNormFactors(exp_study)\n",
      "> et = exactTest(exp_study, pair=c(\"CP-KS-LibL-L7-G_S57_L006_R1_001\", \"CP-KS-LibL-SN1016-G_S26_L006_R1_001\"), dispersion=0.4)\n",
      "> tTags = topTags(et,n=NULL)\n",
      "> result_table = tTags$table\n",
      "> result_table = data.frame(sampleA=\"CP-KS-LibL-L7-G_S57_L006_R1_001\", sampleB=\"CP-KS-LibL-SN1016-G_S26_L006_R1_001\", result_table)\n",
      "> result_table$logFC = -1 * result_table$logFC\n",
      "> write.table(result_table, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L7-G_S57_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results', sep='\t', quote=F, row.names=T)\n",
      "> write.table(rnaseqMatrix, file='1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L7-G_S57_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.count_matrix', sep='\t', quote=F, row.names=T)\n",
      "> source(\"/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\")\n",
      "> pdf(\"1129matrix-trans_counts.isoform.counts.matrix.CP-KS-LibL-L7-G_S57_L006_R1_001_vs_CP-KS-LibL-SN1016-G_S26_L006_R1_001.edgeR.DE_results.MA_n_Volcano.pdf\")\n",
      "> plot_MA_and_Volcano(rownames(result_table), result_table$logCPM, result_table$logFC, result_table$FDR)\n",
      "> dev.off()\n",
      "null device \n",
      "          1 \n",
      "> \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "-note, no biological replicates identified, so setting min reps = 2.\n",
      "Got 24 samples, and got: 25 data fields.\n",
      "Header: CP-KS-LibL-L10-G_S59_L006_R1_001\tCP-KS-LibL-L11-G_S65_L006_R1_001\tCP-KS-LibL-L13-G_S60_L006_R1_001\tCP-KS-LibL-L14-G_S63_L006_R1_001\tCP-KS-LibL-L15-G_S68_L006_R1_001\tCP-KS-LibL-L19-G_S67_L006_R1_001\tCP-KS-LibL-L2-G_S34_L006_R1_001\tCP-KS-LibL-L21-G_S58_L006_R1_001\tCP-KS-LibL-L23-G_S61_L006_R1_001\tCP-KS-LibL-L24-G_S62_L006_R1_001\tCP-KS-LibL-L29-G_S36_L006_R1_001\tCP-KS-LibL-L3-G_S64_L006_R1_001\tCP-KS-LibL-L32-G_S35_L006_R1_001\tCP-KS-LibL-L33-G_S66_L006_R1_001\tCP-KS-LibL-L34-L_S25_L006_R1_001\tCP-KS-LibL-L35-L_S31_L006_R1_001\tCP-KS-LibL-L37-L_S32_L006_R1_001\tCP-KS-LibL-L39-L_S29_L006_R1_001\tCP-KS-LibL-L41-L_S27_L006_R1_001\tCP-KS-LibL-L44-L_S30_L006_R1_001\tCP-KS-LibL-L45-L_S28_L006_R1_001\tCP-KS-LibL-L47-L_S33_L006_R1_001\tCP-KS-LibL-L7-G_S57_L006_R1_001\tCP-KS-LibL-SN1016-G_S26_L006_R1_001\n",
      "Next: TRINITY_DN35250_c0_g1_i1\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\t0\n",
      "\n",
      "$VAR1 = {\n",
      "          'CP-KS-LibL-L44-L_S30_L006_R1_001' => 20,\n",
      "          'CP-KS-LibL-L37-L_S32_L006_R1_001' => 17,\n",
      "          'CP-KS-LibL-L34-L_S25_L006_R1_001' => 15,\n",
      "          'CP-KS-LibL-L35-L_S31_L006_R1_001' => 16,\n",
      "          'CP-KS-LibL-L23-G_S61_L006_R1_001' => 9,\n",
      "          'CP-KS-LibL-L7-G_S57_L006_R1_001' => 23,\n",
      "          'CP-KS-LibL-L32-G_S35_L006_R1_001' => 13,\n",
      "          'CP-KS-LibL-L21-G_S58_L006_R1_001' => 8,\n",
      "          'CP-KS-LibL-L14-G_S63_L006_R1_001' => 4,\n",
      "          'CP-KS-LibL-L15-G_S68_L006_R1_001' => 5,\n",
      "          'CP-KS-LibL-L41-L_S27_L006_R1_001' => 19,\n",
      "          'CP-KS-LibL-L33-G_S66_L006_R1_001' => 14,\n",
      "          'CP-KS-LibL-L2-G_S34_L006_R1_001' => 7,\n",
      "          'CP-KS-LibL-L11-G_S65_L006_R1_001' => 2,\n",
      "          'CP-KS-LibL-L45-L_S28_L006_R1_001' => 21,\n",
      "          'CP-KS-LibL-L47-L_S33_L006_R1_001' => 22,\n",
      "          'CP-KS-LibL-L13-G_S60_L006_R1_001' => 3,\n",
      "          'CP-KS-LibL-L24-G_S62_L006_R1_001' => 10,\n",
      "          'CP-KS-LibL-L10-G_S59_L006_R1_001' => 1,\n",
      "          'CP-KS-LibL-L3-G_S64_L006_R1_001' => 12,\n",
      "          'CP-KS-LibL-L29-G_S36_L006_R1_001' => 11,\n",
      "          'CP-KS-LibL-L39-L_S29_L006_R1_001' => 18,\n",
      "          'CP-KS-LibL-L19-G_S67_L006_R1_001' => 6,\n",
      "          'CP-KS-LibL-SN1016-G_S26_L006_R1_001' => 24\n",
      "        };\n",
      "Contrasts to perform are: $VAR1 = [\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L10-G_S59_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L11-G_S65_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L13-G_S60_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L14-G_S63_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L15-G_S68_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L19-G_S67_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L2-G_S34_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L21-G_S58_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L23-G_S61_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L24-G_S62_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L29-G_S36_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L3-G_S64_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L32-G_S35_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L33-G_S66_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L34-L_S25_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L35-L_S31_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L37-L_S32_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001',\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L39-L_S29_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001',\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L41-L_S27_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001',\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L44-L_S30_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001',\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L45-L_S28_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001',\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L47-L_S33_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ],\n",
      "          [\n",
      "            'CP-KS-LibL-L7-G_S57_L006_R1_001',\n",
      "            'CP-KS-LibL-SN1016-G_S26_L006_R1_001'\n",
      "          ]\n",
      "        ];\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n",
      "Loading required package: edgeR\n",
      "Loading required package: limma\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/run_DE_analysis.pl \\\n",
    "--matrix 1129matrix-trans_counts.isoform.counts.matrix \\\n",
    "--method edgeR \\\n",
    "--dispersion 0.4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "#################################################################################### \r\n",
      "#\r\n",
      "# Required:\r\n",
      "#\r\n",
      "#  --matrix|m <string>       TMM.EXPR.matrix\r\n",
      "#\r\n",
      "# Optional:\r\n",
      "#\r\n",
      "#  -P <float>             p-value cutoff for FDR  (default: 0.001)\r\n",
      "# \r\n",
      "#  -C <float>             min abs(log2(a/b)) fold change (default: 2  (meaning 2^(2) or 4-fold).\r\n",
      "#\r\n",
      "#  --output <float>       prefix for output file (default: \"diffExpr.P${Pvalue}_C${C})\r\n",
      "#\r\n",
      "#\r\n",
      "#\r\n",
      "#\r\n",
      "# Misc:\r\n",
      "#\r\n",
      "#  --samples|s <string>                     sample-to-replicate mappings (provided to run_DE_analysis.pl)\r\n",
      "#\r\n",
      "#  --max_DE_genes_per_comparison <int>    extract only up to the top number of DE features within each pairwise comparison.\r\n",
      "#                                         This is useful when you have massive numbers of DE features but still want to make\r\n",
      "#                                         useful heatmaps and other plots with more manageable numbers of data points.\r\n",
      "#\r\n",
      "#  --order_columns_by_samples_file        instead of clustering samples or replicates hierarchically based on gene expression patterns,\r\n",
      "#                                         order columns according to order in the --samples file.\r\n",
      "#\r\n",
      "#  --max_genes_clust <int>                default: 10000  (if more than that, heatmaps are not generated, since too time consuming)\r\n",
      "#\r\n",
      "#  --examine_GO_enrichment                run GO enrichment analysis\r\n",
      "#       --GO_annots <string>              GO annotations file\r\n",
      "#       --gene_lengths <string>           lengths of genes file\r\n",
      "#\r\n",
      "#       --include_GOplot                  optional: will generate inputs to GOplot and attempt to make a preliminary pdf plot/report for it.\r\n",
      "#\r\n",
      "##############################################################\r\n",
      "\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "! /Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/analyze_diff_expr.pl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[Errno 2] No such file or directory: 'edgeR.96055.dir/'\n",
      "/Users/laura/Documents/roberts-lab/laura-quantseq/results/kallisto\n"
     ]
    }
   ],
   "source": [
    "cd edgeR.96055.dir/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For the below:  \n",
    "-P = p-value cutoff for FDR  \n",
    "-C = min abs(log2(a/b)) fold change (default: 2  (meaning 2^(2) or 4-fold).  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "#################################################################################### \n",
      "#\n",
      "# Required:\n",
      "#\n",
      "#  --matrix|m <string>       TMM.EXPR.matrix\n",
      "#\n",
      "# Optional:\n",
      "#\n",
      "#  -P <float>             p-value cutoff for FDR  (default: 0.001)\n",
      "# \n",
      "#  -C <float>             min abs(log2(a/b)) fold change (default: 2  (meaning 2^(2) or 4-fold).\n",
      "#\n",
      "#  --output <float>       prefix for output file (default: \"diffExpr.P${Pvalue}_C${C})\n",
      "#\n",
      "#\n",
      "#\n",
      "#\n",
      "# Misc:\n",
      "#\n",
      "#  --samples|s <string>                     sample-to-replicate mappings (provided to run_DE_analysis.pl)\n",
      "#\n",
      "#  --max_DE_genes_per_comparison <int>    extract only up to the top number of DE features within each pairwise comparison.\n",
      "#                                         This is useful when you have massive numbers of DE features but still want to make\n",
      "#                                         useful heatmaps and other plots with more manageable numbers of data points.\n",
      "#\n",
      "#  --order_columns_by_samples_file        instead of clustering samples or replicates hierarchically based on gene expression patterns,\n",
      "#                                         order columns according to order in the --samples file.\n",
      "#\n",
      "#  --max_genes_clust <int>                default: 10000  (if more than that, heatmaps are not generated, since too time consuming)\n",
      "#\n",
      "#  --examine_GO_enrichment                run GO enrichment analysis\n",
      "#       --GO_annots <string>              GO annotations file\n",
      "#       --gene_lengths <string>           lengths of genes file\n",
      "#\n",
      "#       --include_GOplot                  optional: will generate inputs to GOplot and attempt to make a preliminary pdf plot/report for it.\n",
      "#\n",
      "##############################################################\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "/Applications/bioinformatics/trinityrnaseq-Trinity-v2.8.3/Analysis/DifferentialExpression/analyze_diff_expr.pl \\\n",
    "--matrix ../1129matrix-trans_counts.isoform.TMM.EXPR.matrix"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
