Check out [this github issue](https://github.com/RobertsLab/project-olympia.oyster-genomic/issues/34#event-1497598684) with information on tissues sent to Katherine for sequencing. She did the library prep. 

The raw RNA-Seq data from Katherine (for gonad transcriptome assembly) saved on Owl:
https://github.com/RobertsLab/project-olympia.oyster-genomic/wiki/RNA-Seq-February-2018

Notes from Katherine: 

_The files you want (that are your pooled gonad samples) are **CP-4Spl_S11_L004_R1_0343.fastq.gz,  CP-4Spl_S11_L004_R2_0343.fastq.gz,  CP-4Spl_S11_L004_R1_0348.fastq.gz,  CP-4Spl_S11_L004_R2_0348.fastq.gz.** That is the 1st and 2nd read from 2 lanes of sequencing. The other files are from my CA, OR, and British Columbia populations.

Here is the link to where the QuantSeq raw files are being uploaded: http://owl.fish.washington.edu/wetgenes/201803_QuantSeq_Raw/ 

[This link at UC Davis](http://dnatech.genomecenter.ucdavis.edu/tag-seq-gene-expression-profiling/) gives a good overview of what Tag-Seq (aka QuantSeq) is and how it differs from traditional RNA seq. The kit I used to make the libraries is from Lexogen, called [QuantSeq 3' mRNA-Seq](https://www.lexogen.com/quantseq-3mrna-sequencing/). It's a good idea to be familiar with how these data differ from traditional RNA-Seq as the pipeline is different than many RNA-Seq tutorials. For my analysis, I use a lot of [code from Misha Matz's lab](https://github.com/z0on/tag-based_RNAseq) at UT Austin. They were involved in designing TagSeq._
